cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -250,10 +248,10 @@ To detect misconfigurations in Docker:
 
 By default, the Docker daemon is configured to have a base logging level of 'info', and if this is not the case: set the Docker daemon log level to 'info'. Rationale: Setting up an appropriate log level, configures the Docker daemon to log events that you would want to review later. A base log level of 'info' and above would capture all logs except the debug logs. Until and unless required, you should not run docker daemon at the 'debug' log level.
 
-To configure the log level in docker-compose:
+To configure the log level in Docker Compose:
 
 ```bash
-docker-compose --log-level info up
+docker compose --log-level info up -d
```

Comment:
Why add `-d`?

Reply:
Added detached flag for a slightly more typical example, but taking a second look at that section the whole command isn't actually relevant.